sessionctx/variable/session.go

```diff
@@ -1503,6 +1503,14 @@ type SessionVars struct {
 	// OptimizerFixControl control some details of the optimizer behavior through the tidb_opt_fix_control variable.
 	OptimizerFixControl map[uint64]string
 
+	// EnableRemovePartitioning for ALTER TABLE t REMOVE PARTITIONING
+	EnableRemovePartitioning bool
```

Comment:
Could we remove those variables?

Reply:
Yes, I just had them there to remind that TiFlash and TiCDC is not yet supporting these ALTER commands.

Reply:
Removed.